examples/basic_openshift/conjur-java-api-example.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
```

Comment:
If the example works for Kubernetes and not just for OpenShift, I would change the folder name and all relevant references to Kubernetes, because it's a more generic name.